core-web/libs/block-editor/src/lib/shared/components/suggestions/suggestion-icons.ts

```diff
@@ -23,3 +23,9 @@ export const codeIcon =
 
 export const lineIcon =
     'data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzIiIGhlaWdodD0iMzIiIHZpZXdCb3g9IjAgMCAzMiAzMiIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHJlY3QgeD0iNiIgeT0iMTUiIHdpZHRoPSIyMCIgaGVpZ2h0PSIyIiBmaWxsPSJibGFjayIvPgo8L3N2Zz4K';
+
+export const imageIcon =
```

Comment:
we also can use the icon name directly, no need to create a base64, right ?

Reply:
You are right! thank you!